### PR TITLE
Add reset filter button in LessonFilter

### PR DESCRIPTION
**Pull request by kristandre**

### CODE REVIEW

**Comment by kristandre** [comment]
Solves issue #30

File: src/components/Filter/LessonFilter.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, {PropTypes} from 'react';
 import FilterGroup from './FilterGroup';
 import styles from './LessonFilter.scss';
 import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import {Button} from 'react-bootstrap';
```
Comment (NorwegianKiwi) [inline src/components/Filter/LessonFilter.js]: import from react-bootstrap/lib/...

**Comment by NorwegianKiwi** [comment]
Mangler tester for RESET_FILTER

File: src/components/Filter/LessonFilter.js [hunk 2]
```diff
@@ -22,6 +23,8 @@ const LessonFilter = React.createClass({
           </div>
           <div className={'panel-body ' + styles.noPaddingTop}>
             {filterGroups}
+            <br/>
+            <Button block onClick={() => this.props.resetFilter()}>Fjern filter</Button>
```
Comment (NorwegianKiwi) [inline src/components/Filter/LessonFilter.js]: resetFilter må inn inn i propTypes

**Pull request merged**